### ISSUE TITLE
Sqlite Provider InvalidCastException

### ISSUE THREAD

**Issue by MarkZither** [MiniProfiler/dotnet#578]
When using the Sqlite Provider i encountered the following error.

```
An unhandled exception occurred while processing the request.
InvalidCastException: Unable to cast object of type 'System.Int64' to type 'System.Double'.
Deserialize0693dad1-08a7-4b56-b158-09ed5699f3a7(IDataReader )

DataException: Error parsing column 5 (StartMilliseconds=120 - Int64)
Dapper.SqlMapper.ThrowDataException(Exception ex, int index, IDataReader reader, object value) in SqlMapper.cs, line 3665
```

This is similar to the final message in the PR #221 

It seems to stem from the dynamic [datatypes of Sqlite](https://www.sqlite.org/datatype3.html).

**Comment by MarkZither** [comment]
Is my change ok to be accepted?

**Comment by meum** [comment]
I also ran into this one today, would be great to get the PR merged

**Comment by NickCraver** [comment]
I happen to be working on this after a long break just this week - this will be in a 4.3 release soon :)

**Comment by NickCraver** [comment]
Going to close this our since #582 is merged in - look out for a 4.3 release on NuGet soon, but already on MyGet now :)
